packages/docusaurus-theme-classic/src/theme/MDXComponents/index.tsx

```diff
@@ -12,8 +12,8 @@ import MDXA from '@theme/MDXComponents/A';
 import MDXPre from '@theme/MDXComponents/Pre';
 import MDXDetails from '@theme/MDXComponents/Details';
 import MDXHeading from '@theme/MDXComponents/Heading';
-import MDXUl from '@theme/MDXComponents/Ul';
 import MDXImg from '@theme/MDXComponents/Img';
+import './task-list.css';
```

Comment:
MDXUl is a better place to import the CSS there. I'd like to avoid such imports in MDXComponents and just keep it as a simple map

BTW if MDX injets a `.contains-task-list`, it does not necessarily mean that we have to apply it, and we can keep using scoped CSS modules